src/nginxconfig/templates/global_sections/logging.vue

```diff
@@ -51,17 +76,44 @@ THE SOFTWARE.
             <div class="field-body">
                 <div class="field">
                     <div :class="`control${errorLogChanged ? ' is-changed' : ''}`">
+                        <div class="checkbox">
+                            <PrettyCheck v-model="errorLog" class="p-default p-curve p-fill p-icon">
+                                {{ $t('common.enable') }}
+                            </PrettyCheck>
+                        </div>
+                    </div>
+                    <div v-if="$props.data.errorLog.computed" class="control field is-horizontal is-expanded">
                         <input
-                            v-model="errorLog"
+                            v-model="errorLogPath"
                             class="input"
                             type="text"
-                            :placeholder="$props.data.errorLog.default"
+                            :placeholder="$props.data.errorLogPath.default"
                         />
                     </div>
```

Comment:
Ditto on all three thoughts here (is-changed indicator, watcher for the default computed, parent field alignment)